Build issues with windows 10 for Debug configuration

Followed everything from the [wiki page](https://github.com/OpenXRay/xray-16/wiki/%5BEN%5D-How-to-build-and-setup-on-Windows), cloned with submodules, installed the required MSVC packages (windows sdk, nuget, universal crt)

Building Mixed, Release or Release Master Gold works without problem just Debug fails with the following messages:

```
Build started...
1>------ Build started: Project: BuildVm, Configuration: Debug x64 ------
2>------ Build started: Project: xrCore, Configuration: Debug x64 ------
1>BuildVm.vcxproj -> D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\BuildVm.exe
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: The command "set ALL_LIB=lib_base.c lib_math.c lib_bit.c lib_string.c lib_table.c lib_io.c lib_os.c lib_package.c lib_debug.c lib_jit.c lib_ffi.c
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: 
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: cd D:\dev\xray-16\src\..\Externals\LuaJIT\src
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\BuildVm" -m peobj -o "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\lj_vm_x64.obj"
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\BuildVm" -m bcdef -o "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\lj_bcdef.h" %ALL_LIB%
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\BuildVm" -m ffdef -o "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\lj_ffdef.h" %ALL_LIB%
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\BuildVm" -m libdef -o "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\lj_libdef.h" %ALL_LIB%
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\BuildVm" -m recdef -o "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\lj_recdef.h" %ALL_LIB%
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\BuildVm" -m folddef -o "D:\dev\xray-16\Externals\LuaJIT-proj\intermediate\x64\lj_folddef.h" lj_opt_fold.c
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073:       
1>C:\Program Files (x86)\Microsoft Visual Studio\2019\Community\MSBuild\Microsoft\VC\v160\Microsoft.CppCommon.targets(155,5): error MSB3073: :VCEnd" exited with code -1073741502.
1>Done building project "BuildVm.vcxproj" -- FAILED.
2>git found
2>xrCore.cpp
2>Bone.obj : warning LNK4075: ignoring '/EDITANDCONTINUE' due to '/INCREMENTAL:NO' specification
2>   Creating library D:\dev\xray-16\src\..\build\lib\x64\Debug\xrCore.lib and object D:\dev\xray-16\src\..\build\lib\x64\Debug\xrCore.exp
2>LINK : warning LNK4098: defaultlib 'LIBCMT' conflicts with use of other libs; use /NODEFAULTLIB:library
2>LINK : warning LNK4217: symbol '?Fail@xrDebug@@SA?AW4AssertionResult@@AEA_NAEBVErrorLocation@@PEBD222@Z (public: static enum AssertionResult __cdecl xrDebug::Fail(bool &,class ErrorLocation const &,char const *,char const *,char const *,char const *))' defined in 'xrDebug.obj' is imported by 'xrMiscMath.lib(vector.obj)' in function '"public: struct _vector3<float> & __cdecl _vector3<float>::normalize(struct _vector3<float> const &)" (?normalize@?$_vector3@M@@QEAAAEAU1@AEBU1@@Z)'
2>LINK : warning LNK4286: symbol '?Fail@xrDebug@@SA?AW4AssertionResult@@AEA_NAEBVErrorLocation@@PEBD222@Z (public: static enum AssertionResult __cdecl xrDebug::Fail(bool &,class ErrorLocation const &,char const *,char const *,char const *,char const *))' defined in 'xrDebug.obj' is imported by 'xrMiscMath.lib(matrix.obj)'
2>LINK : warning LNK4217: symbol '?randF@CRandom@@QEAAMM@Z (public: float __cdecl CRandom::randF(float))' defined in 'Model.obj' is imported by 'xrMiscMath.lib(vector.obj)' in function '"public: struct _vector3<float> & __cdecl _vector3<float>::random_dir(class CRandom &)" (?random_dir@?$_vector3@M@@QEAAAEAU1@AEAVCRandom@@@Z)'
2>LINK : warning LNK4217: symbol '?randFs@CRandom@@QEAAMM@Z (public: float __cdecl CRandom::randFs(float))' defined in 'Model.obj' is imported by 'xrMiscMath.lib(vector.obj)' in function '"public: struct _vector3<float> & __cdecl _vector3<float>::random_point(struct _vector3<float> const &,class CRandom &)" (?random_point@?$_vector3@M@@QEAAAEAU1@AEBU1@AEAVCRandom@@@Z)'
2>xrCore.vcxproj -> D:\dev\xray-16\bin\x64\Debug\xrCore.dll
2>Done building project "xrCore.vcxproj".
3>------ Skipped Build: Project: dummy, Configuration: Debug x64 ------
3>Project not selected to build for this solution configuration 
4>------ Skipped Build: Project: xrWeatherEngine, Configuration: Debug x64 ------
4>Project not selected to build for this solution configuration 
5>------ Skipped Build: Project: xrWeatherEditor, Configuration: Debug x64 ------
5>Project not selected to build for this solution configuration 
6>------ Skipped Build: Project: Common, Configuration: Debug x64 ------
6>Project not selected to build for this solution configuration 
========== Build: 1 succeeded, 1 failed, 55 up-to-date, 4 skipped ==========
```

--- a/src/xray_build.props
+++ b/src/xray_build.props
@@ -49,7 +49,6 @@
   <!-- Common debug settings -->
   <PropertyGroup Condition="$(Configuration.Contains('Debug'))" Label="Configuration">
     <UseDebugLibraries>true</UseDebugLibraries>
-	<EnableASAN>true</EnableASAN>
     <LinkIncremental>false</LinkIncremental>
   </PropertyGroup>
 

